ui/app/src/utils/itemActions.ts

```diff
@@ -965,7 +965,7 @@ export const itemActionDispatcher = ({
     case 'schedulePublish':
     case 'requestPublish': {
       const schedulingMap = {
-        approvePublish: null,
+        approvePublish: undefined,
```

Comment:
When setting the `approvePublish` value to null, the default value in the component was being overwritten, causing the scheduling calculation to be wrong. If `undefined`, the scheduling it defaulted to 'now' (custom schedules are still working properly).